lib/applications/schema.graphql

```diff
@@ -124,3 +126,43 @@ input CreateApplicationInput {
 type CreateApplicationResult {
   ok: Boolean!
 }
+
+input ApplicationsFilter {
+  order: [[String!]]
+  permitType: PermitType
+  requestType: String
+  status: ApplicationStatus
+  search: String
+  limit: Int
+  offset: Int
+}
+
+type ApplicationProcessing {
+  id: Int
+  status: ApplicationStatus
+  appNumber: Int
+  appHolepunched: Boolean
+  walletCardCreated: Boolean
+  invoiceNumber: Int
+  documentUrls: [String]
+  appMailed: Boolean
+  applicationId: Int
+  createdAt: Date
+  updatedAt: Date
+  application: Application
+}
+
+type ApplicationsFilterResult{
+  firstName: String!
+  lastName: String!
+  id: ID!
+  createdAt: Date!
+  applicationProcessing: ApplicationProcessing
+  isRenewal: Boolean!
+  permitType: PermitType!
+}
```

Comment:
I don't think we need a custom return type when filtering applications, `Application` should work just fine, no?
applicationProcessing should also probably be added to the Application type as well

Reply:
Yeah you're right, good catch